lib/shaman/tryout_apps/client.rb

```diff
@@ -0,0 +1,25 @@
+module Shaman
+  module TryoutApps
+    class Client
+      def initialize(base_uri)
+        @base_uri = base_uri
+      end
+
+      def list_projects(params)
```

Comment:
Can you check NewRelic data to find the 99th percentile response time? I'd like us to add a specific timeout

Reply:
Let's tackle it later then, I'm not confident that all the other actions in here would fall within a generic X second timeout, especially the release creation ones.